index.js

```diff
@@ -21,4 +22,28 @@ function hideMenu() {
 
 document.getElementById('hamburger-id').addEventListener('click', showMenu);
 document.getElementById('cancel-id').addEventListener('click', hideMenu);
-document.querySelector('.menu-list').addEventListener('click', hideMenu);
\ No newline at end of file
+document.querySelector('.menu-list').addEventListener('click', hideMenu);
+
+// popup
+const modal = document.getElementById('myModal');
+const raiseModal = document.querySelectorAll('.raise-modal');
+raiseModal.forEach((trigger) => {
+  trigger.addEventListener('click', () => {
+    modal.style.display = 'block';
+  });
+});
```

Comment:
I noticed that you created a static modal in your HTML that you're fetching and displaying in this function.

- [ ] Kindly create an array of projects holding objects of the different projects. It's required that you have that array in a `.js` file, loop through that array, and populate each modal upon clicking with its details in the object. Kindly work on that. 👍 

### Hint:
```mdx
const projects = [{
    name: "Name of the project",
    description: "description of the project",
    image: "link_to_image.jpg",
    technologies: ["HTML", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
``` 
- Please use that format to store the popup details for every project and pass it to the modal when clicked. 👍